### ISSUE TITLE
Dependencies deprecation warnings

### ISSUE THREAD

**Issue by madhav-vl** [JCMais/node-libcurl#317]
Could you please resolve below dependant package deprecation warning

> node-pre-gyp@0.15.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future

**Comment by JCMais** [comment]
This will be fixed this week. Thanks for reporting.

**Comment by greenchapter** [comment]
@JCMais is there any update, because actually `node-libcurl@2.3.3` breaks on Apple Silicon Macs

**Comment by JCMais** [comment]
@greenchapter there is a build issue at the moment that I'm trying to fix, pre-release should go out as soon as I get time to fix it. I will try to find time this week.

**Comment by greenchapter** [comment]
> @greenchapter there is a build issue at the moment that I'm trying to fix, pre-release should go out as soon as I get time to fix it. I will try to find time this week.

It looks like there is a missing file
```
npm ERR! node-pre-gyp info check checked for "/Users/xxx/Workshop/github/packages/node_modules/node-libcurl/lib/binding/node_libcurl.node" (not found)
```

**Comment by JCMais** [comment]
This has been fixed with `2.3.4`
